### DISCUSSION TITLE
After installing in docker, I didn't find the backend username and password information.

### DISCUSSION THREAD

**Discussion by tippytea** [mediacms-io/mediacms#539] [Q&A]
The mediacms website is working fine. But there is no backend username and password, how do I find the backend administrator username and password?

https://github.com/mediacms-io/mediacms/discussions/81

I was trying to solve the problem with the above answer, but I found that I did not find the "bin" folder in the /home/mediacms.io directory that I installed in docker.

**Comment by mgogoulos** [comment]
try to find it on the docker logs, eg `docker logs mediacms|grep password` or something like that. 
If you want to login through the python/django shell to change it (as on the link https://github.com/mediacms-io/mediacms/discussions/81) you have to bash into the container first...

docker ps , check the id of the container mediacms, and then 
docker exec -it ID_OF_CONTAINER bash
and you are in the python/django shell

**Reply by tippytea** [reply]
When I execute this command "docker logs mediacms|grep password", it prompts: Error: No such container: mediacms

**Reply by tippytea** [reply]
https://i.imgur.com/UWjUZDa.png
